securityscanutils/cli/exampleVersionImageConstraints.txt

```diff
@@ -0,0 +1,4 @@
+# comments are skipped, as are empty lines
+
+constraint-a , image1, image2, image3
```

Comment:
the spaces will be included in the image names, which may cause errors here. Do you strip the spaces before processing the csv?

Reply:
Also can you include an actual example of images and a constraint, like
```csv
>=v1.6.0, gloo-ee, rate-limit-ee, extauth-ee
```

Reply:
Whitespace is trimmed here: https://github.com/solo-io/go-utils/pull/468/files#diff-f92cc044bef1930064629dba7b446d6e84fabfb02082830689bcc858f6e797dcR398

Reply:
Added a commented real example constraint line.